verification/cbmc/proofs/aws_byte_cursor_read/aws_byte_cursor_read_harness.c

```diff
@@ -39,7 +39,7 @@ void aws_byte_cursor_read_harness() {
             assert_byte_from_buffer_matches(cur.ptr, &old_byte_from_cur);
         }
     } else {
-        assert(cur.ptr == old_cur.ptr + length);
+        assert(AWS_IMPLIES(old.ptr != NULL, cursor.ptr == old.ptr + len));
```

Comment:
Hmm, it seems to me like we are losing something here after this change.
Now we don't seem to check anything if `old_cur.ptr` is `NULL`.

How about:
```c
if (old_cur.ptr != NULL) {
    assert(cur.ptr == old_cur.ptr + length);
} else {
    assert(cur.ptr == NULL);
}
```

Reply:
Please double check the change.